bitcoin/src/blockdata/weight.rs

```diff
@@ -173,8 +173,8 @@ mod tests {
 
     #[test]
     fn from_vb_unchecked() {
-        let vb = Weight::from_vb_unchecked(1);
-        assert_eq!(Weight(4), vb);
+        let wu = Weight::from_vb_unchecked(1);
+        assert_eq!(Weight(4), wu);
```

Comment:
I won't block this since it's not important but in the future please refrain from naming variables with units if the type already encodes the unit. If this was called `weight` from the beginning the renaming wouldn't be needed.

Reply:
Thanks for the tip.  May as well get it right this PR.  changed this to `weight` instead of `wu` https://github.com/rust-bitcoin/rust-bitcoin/compare/31392f27e44be3eefdb827f4d2a522bd59eceeb2..a4934bf9d230e6365e3777dc260d68e17663dfc7